activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -41,8 +41,9 @@ def set_fixture_class(class_names = {})
       def fixtures(*fixture_set_names)
         if fixture_set_names.first == :all
           raise StandardError, "No fixture path found. Please set `#{self}.fixture_path`." if fixture_path.blank?
-          fixture_set_names = Dir["#{fixture_path}/{**,*}/*.{yml}"].uniq
-          fixture_set_names.map! { |f| f[(fixture_path.to_s.size + 1)..-5] }
+          fixture_set_names = Dir[::File.join(fixture_path, "{**,*}/*.{yml}")].uniq
+          fixture_set_names.reject! { |f| f.starts_with?(file_fixture_path.to_s) } if try(:file_fixture_path)
```

Comment:
Why we need to`try`? `FileFixtures` is inside `ActiveSupport::TestCase`

Reply:
Can we assume `ActiveRecord::TestFixtures` will always be included in `ActiverSupport::TestCase`? I thought not, but I could be accounting for really niche edge cases.

Reply:
We can. It is always included in the rails code base.